assets/application.json

```
@@ -40,6 +40,12 @@
     "products/variations/price": []
   },
   "admin_settings": {
+    "disable": {
+      "type": "boolean",
+      "default": false,
+      "title": "Desabilitar cartão de crédido",
+      "description": "Desabilitar cartão de crédito via MP"
```

```suggestion
      "title": "Desabilitar cartão de crédido"
```

Acho que não precisa repetir não, a descrição é opcional e mais pra quando tem que ter um help text maior mesmo, e diferente do título...

---

Beleza, valeu